ncm-metaconfig/src/main/metaconfig/conntrackd/pan/schema.pan

```diff
@@ -0,0 +1,117 @@
+declaration template metaconfig/conntrackd/schema;
+
+include 'pan/types';
+
+@documentation{
+    Conntrackd config
+}
+
+@documentation{
+    There are 3 main synchronization modes or protocols: NOTRACK, ALARM and FTFW.
+}
+type conntrackd_service_sync_mode = {
+    'type' : choice('FTFW', 'ALARM', 'NOTRACK,') = 'FTFW'
+    'DisableExternalCache' ? boolean
+    'CommitTimeout' ? long
+    'PurgeTimeout' ? long
```

Comment:
Any bounds for these values? The man page suggests these are seconds, so at least `(0..)`?